envs/annotation_scATAC.yml

```diff
@@ -0,0 +1,52 @@
+name: bio96
+channels:
+  - anaconda
+  - bioconda
+  - r
+  - conda-forge
+  - defaults
+dependencies:
+  - blas=1.0=mkl
+  - ca-certificates=2019.10.16=0
+  - certifi=2019.9.11=py37_0
+  - intel-openmp=2019.5=281
+  - libedit=3.1.20181209=hc058e9b_0
+  - libffi=3.2.1=h4deb6c0_3
+  - libgcc-ng=9.1.0=hdf63c60_0
+  - libgfortran-ng=7.3.0=hdf63c60_0
+  - libstdcxx-ng=9.1.0=hdf63c60_0
+  - mkl=2019.5=281
+  - mkl-service=2.3.0=py37he904b0f_0
+  - mkl_fft=1.0.15=py37ha843d7b_0
+  - mkl_random=1.1.0=py37hd6b4f25_0
+  - ncurses=6.1=he6710b0_1
+  - numpy=1.17.3=py37hd14ec0e_0
+  - numpy-base=1.17.3=py37hde5b4d6_0
+  - openssl=1.1.1=h7b6447c_0
+  - pandas=0.25.3=py37he6710b0_0
+  - pip=19.3.1=py37_0
+  - python=3.7.5=h0371630_0
+  - python-dateutil=2.8.1=py_0
+  - pytz=2019.3=py_0
+  - readline=7.0=h7b6447c_5
+  - setuptools=42.0.1=py37_0
+  - six=1.13.0=py37_0
+  - sqlite=3.30.1=h7b6447c_0
+  - tk=8.6.8=hbc83047_0
+  - wheel=0.33.6=py37_0
+  - xz=5.2.4=h14c3975_4
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - bio96==0.5.1
+    - colorcet==2.0.2
+    - cycler==0.10.0
+    - docopt==0.6.2
+    - kiwisolver==1.1.0
+    - matplotlib==3.1.2
+    - nonstdlib==1.12.0
+    - param==1.9.2
+    - pyct==0.4.6
+    - pyparsing==2.4.5
+    - toml==0.10.0
+prefix: /home/jsmits/anaconda3/envs/bio96
+
```

Comment:
Actually since I am gonna write a seperate annotation script for the lab (this is easier because it would break on public data), this env has become obsolete. Ill remove it and update the pull request.